projects/packages/codesniffer/Jetpack/ruleset.xml

```diff
@@ -69,4 +69,9 @@
 		<exclude name="WordPress.DB.SlowDBQuery"/>
 	</rule>
 
+	<!-- This rule, while useful, is not one we'd want to run automatically in a pre-commit hook. -->
+	<rule ref="WordPress.WP.CapitalPDangit.Misspelled">
+		<exclude name="WordPress.WP.CapitalPDangit.Misspelled" />
```

Comment:
This would stop the rule overall. Do we still want the rule, but simply not autofix? 

If the later, we could `exclude=WordPress.WP.CapitalPDangit` for `phpcbf` on the pre-commit hook (and, if desired, add the similar thing to `composer.json` for the script there).

I don't see a way to add, via config, to allow a rule, but only exclude it from autofix.

Reply:
> Do we still want the rule, but simply not autofix?

This is what I was wondering as well:

https://github.com/Automattic/jetpack/pull/24363#issuecomment-1126194355

Reply:
I'd be okay with that. That does stop us from committing, but I think that's better than autofixing:

```
 git ci
> printf '\e[30;43m`composer php:lint` now lints. If you want to run phpcs, use `composer phpcs:lint`.\e[0m\n' '--files' 'projects/plugins/jetpack/modules/custom-post-types/nova.php'
`composer php:lint` now lints. If you want to run phpcs, use `composer phpcs:lint`.
> tools/parallel-lint.sh '--files' 'projects/plugins/jetpack/modules/custom-post-types/nova.php'
PHP 8.0.18 | 10 parallel jobs
.                                                            1/1 (100 %)


Checked 1 files in 0 seconds
No syntax error found
> Composer\Config::disableProcessTimeout
> phpcs -p -s '--standard=./.phpcs.config.xml,PHPCompatibilityWP' '--runtime-set' 'jetpack-filter-perdir-file' '.phpcs.dir.phpcompatibility.xml' 'projects/plugins/jetpack/modules/custom-post-types/nova.php'
. 1 / 1 (100%)


Time: 124ms; Memory: 20MB


No fixable errors were found

Time: 349ms; Memory: 28MB

> Composer\Config::disableProcessTimeout
> phpcs -p -s '--runtime-set' 'ignore_warnings_on_exit' '1' 'projects/plugins/jetpack/modules/custom-post-types/nova.php'
W 1 / 1 (100%)



FILE: ...pack/projects/plugins/jetpack/modules/custom-post-types/nova.php
----------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
----------------------------------------------------------------------
 112 | WARNING | [x] Please spell "WordPress" correctly. Found 1
     |         |     misspelling(s): wordpress
     |         |     (WordPress.WP.CapitalPDangit.Misspelled)
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------

Time: 349ms; Memory: 28MB

Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Package operations: 0 installs, 0 updates, 0 removals
Generating autoload files
2 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
> Composer\Config::disableProcessTimeout
> phpcs-changed -s --git 'projects/plugins/jetpack/modules/custom-post-types/nova.php'

FILE: projects/plugins/jetpack/modules/custom-post-types/nova.php
-----------------------------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
-----------------------------------------------------------------------------------------------
 112 | WARNING | Please spell "WordPress" correctly. Found 1 misspelling(s): wordpress (WordPress.WP.CapitalPDangit.Misspelled)
-----------------------------------------------------------------------------------------------
Script phpcs-changed -s --git handling the phpcs:changed event returned with error code 1
COMMIT ABORTED: The linter reported some problems. If you are aware of them and it is OK, repeat the commit command with --no-verify to avoid this check.
But please don't. Code is poetry.


husky - pre-commit hook exited with code 1 (error)
```

Reply:
Yeah, I like that. If there's a misspelling, we can fix it manually. If it is intentional, we can phpcs:ignore.